### ISSUE TITLE
Please improve opening old database files error note

### ISSUE THREAD

**Issue by Jaz-z** [keepassxreboot/keepassxc#6950]
## Summary
An error message already exists to import old files but it sometimes dose not work and sometimes shows only momentarily not noticeable enough for some people - suggestion for improvement. 

Operating System: Kubuntu 20.04
KDE Plasma Version: 5.18.5
KDE Frameworks Version: 5.68.0
Qt Version: 5.12.8

## The issue
On trying to open an old database file .kdb with KeePassX 2.0.3 it shows the following error and guide text in a stand alone dialogue box that stays open until closed by user:
----
Error:
Unable to open the database.
The Selected file is an old KeePass 1 database (.kdb).
You can import it by clicking on Databse > ‘Import KeePass1 database’.
This is a one-way migration. You won’t be able to open the imported database with the old KeePassX 0.4 version.
---

KeePassXC 2.4.3 sometimes shows a similar error message but it opens momentarily at the top of the main KeePassXC window and closes quickly. (there have been times that it did not show this or I didn't see it).
If the user is using a dark theme in the computer the above message cannot be seen well enough to be read in KeePassXC.

In addition,  KeePassX 2.0.3 tries to open the .kdb file on first attempt, if the user, uses right click on the file and chooses from the list of applications to open the database with  KeePassX 2.0.3. 

In comparison KeePassXC did not open the file on the first attempt, however it opened it by using KeePassXC menu to open database, after this exercise, on following attempts to open old file by right click, it recognised  it and tried to open.

Could you please separate the error message into a separate dialogue box and  add this guide note / message to KeePassXC: ‘after import use ‘save as’ to convert the file to the latest format .kdbx’.

## Context 
I tried to open old file .kdb with KeePassXC by right click it did not do anything and I did not see an error message so I installed KeePassX to open the old file which showed me an error message and how to import the old file. After this I checked KeePassXC only to find out the long way that it also had the same import tool. 
On later attempts I found that it does sometimes show a short lived error message which in my computer themes shows as red and difficult to see - I may have missed it the first time. Not suitable for older people or people with sight problems, the KeePassX message in comparison can be seen easily and stays open until the user takes note and closes it.

See photos below.

![keepassX example](https://user-images.githubusercontent.com/73505939/134804643-2640c13c-3441-4a8e-a641-083b4be228b3.png)
![keePassXC example](https://user-images.githubusercontent.com/73505939/134804644-7dcf8668-88d2-4c30-bde0-5baa2bd47682.png)

**Comment by phoerious** [comment]
Why do you even want to open KDB files? That is a one-time import process. KDB has been deprecated for years and shouldn't be used anymore (even though KeePass1 is still supported for whatever reason).

**Comment by Jaz-z** [comment]
I suddenly had to check an information from years ago that I had deleted from new files and found it in an old file that I had kept 'just in case' but had not gone back to for years, so it remained in the old file format. Obviously I have now converted it to the new format.
Backward compatibility is always useful. It is good KeePass1 is still supported, even though rarely needed, people may suddenly have a need in similar situations which is quite important to them.
